entropylab/api/in_process_param_store.py

```diff
@@ -116,20 +116,19 @@ def commit(self, label: Optional[str] = None) -> str:
         with self.__lock:
             if not self.__is_dirty:
                 return self.__base_commit_id
-            metadata = self.__generate_metadata(label)
-            if self.__is_in_memory_mode:
-                params = _deep_copy(self.toDict())
```

Comment:
Please note that I have removed this method of deep copying the `params` dict because Munch's `toDict()` method makes it redundant.